src/main/java/cohen/morseCode/MorseCode.java

```diff
@@ -0,0 +1,76 @@
+package cohen.morseCode;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MorseCode
+{
+    private static final String[] alphabet = new String[]{" ", "A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M", "N",
+            "O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z"};
+    private final ArrayList<String> english = new ArrayList<>();
+
+    private static final String[] code = new String[]
+            {"       ", "*-", "-***", "-*-*", "-**", "*", "**-*", "--*", "****", "**", "*---", "-*-", "*-**", "--", "-*", "---",
+                    "*--*", "--*-", "*-*", "***", "-", "**-", "***-", "*--", "-**-", "-*--", "--**"};
+    private final ArrayList<String> morseCode = new ArrayList<>();
+
+    public MorseCode()
+    {
+        english.addAll(List.of(alphabet));
+        morseCode.addAll(List.of(code));
```

Comment:
Can you figure out how to do this without needing the `alphabet` and `code` arrays?